src/pybind/mgr/dashboard_v2/tests/helper.py

```diff
@@ -100,3 +104,7 @@ def _rbd_cmd(cls, cmd):
         _cmd = ['rbd']
         _cmd.extend(cmd)
         return cls._cmd(_cmd)
+
+
+skip_if_loaded_by_mgr = unittest.skipIf(bool(os.environ.get('SKIP_LOADED_BY_MGR', False)),
```

Comment:
This seems to suggest that we will be skipping unit tests if we were "loaded by the mgr"? Why?